pkg/stanza/operator/input/windows/input.go

```diff
@@ -232,45 +233,23 @@ func (i *Input) read(ctx context.Context) int {
 
 // processEvent will process and send an event retrieved from windows event log.
 func (i *Input) processEvent(ctx context.Context, event Event) {
-	remoteServer := i.remote.Server
-
-	if i.raw {
-		if len(i.excludeProviders) > 0 {
```

Comment:
The exclusion of providers is being dropped. I'm not familiar with the history here, I just that this current code is a bit wasteful: it renders "simple" to look at the provider and then it likely renders the event as raw, so in the end it generates the same XML that it could used from the simple... Anyway, since EventRaw is being removed this need to be reviewed and updated.

Reply:
I don't think I dropped it, but did refactor it so that we do it the same regardless of other settings. That said, I'm having another look at the behavior. (See other comment)

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34755 is now tracking this